w10h01/test/pgdp/trials/TrialOfTheDreamsTest.java

```diff
@@ -0,0 +1,87 @@
+package w10h01.test.pgdp.trials;
+
+import java.util.function.Function;
+import java.security.SecureRandom;
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+import java.time.Duration;
+import static org.junit.jupiter.api.Assertions.assertTimeoutPreemptively;
+
+public class TrialOfTheDreamsTest {
+
+	// change for harder decoding
+	int keyLen = 16;
+	// change to test expected code duration in seconds (Artemis times out after 5s)
+	int expectedTime = 5;
+	// change to test a specific key with "test_not_random_LockPick"
+	byte[] expectedKey = new byte[] { 40, 41, 42, 43 };
+
```

Comment:
```suggestion
	// Use this seed in Random
	static final int seed = 420;

```
See @felixhauptmann's comment